### ISSUE TITLE
emacs-26.1-mac-7.1 downloaded has shasum as indicated to match emacs-26.1-mac-7.1-spacemacs-icon

### ISSUE THREAD

**Issue by jfbu** [railwaycat/homebrew-emacsmacport#145]
I downloaded [from release page](https://github.com/railwaycat/homebrew-emacsmacport/releases) via [this link](https://s3.amazonaws.com/emacs-mac-port/emacs-26.1-mac-7.1.zip). Here is computed shasum of downloaded archive:

```console
$ shasum -a 256 emacs-26.1-mac-7.1.zip 
579c0534f4c3c37303c6b50b757b6c88264555a6f5dec0a5d7562dc417d193ff  emacs-26.1-mac-7.1.zip
```

i.e. it is the shasum

```
579c0534f4c3c37303c6b50b757b6c88264555a6f5dec0a5d7562dc417d193ff
```

given for the [emacs-26.1-mac-7.1-spacemacs-icon](https://s3.amazonaws.com/emacs-mac-port/emacs-26.1-mac-7.1-spacemacs-icon.zip) download.

Actually the shasums are permuted

```
$ echo "efd9e32fa8d71a6f92758bb83a82c99df879188b7960c717eb1c5d6d90b9fbd2  emacs-26.1-mac-7.1-spacemacs-icon.zip" | shasum -a 256 -c
emacs-26.1-mac-7.1-spacemacs-icon.zip: OK
```

**Comment by railwaycat** [comment]
Thanks to point it out! That's my mistake..I have updated the release page. 

I'm thinking of remove shasums from release page, since developer signature for bundles should already gives a quite strong protection from been modified.

**Comment by jfbu** [comment]
@railwaycat ok, do you mean simply double-clicking in Finder on downloaded zip is safe enough? I can then drag folder to Applications with no worries?

**Comment by railwaycat** [comment]
Hmm, not exactly. 

If you download from release page and have

"Security & Privacy"
  -> General
    -> Allow apps downloaded from: [x] App Store and identified developers checked

in your System Preferences and there's no warning like "Emacs.app is from an unidentified developer" something when you double click, usually means it's safe. 

And you can always verify the app by `codesign -dvv Emacs.app` and check if it has the following lines:

>Authority=Developer ID Application: Xin Xu (3L5CAL23RF)
Authority=Developer ID Certification Authority
Authority=Apple Root CA

It's not less security than shasum but richer in information.

BTW, bundles downloaded by brew cask will still check shasum.

**Comment by jfbu** [comment]
Yes I have this security setting on. I drag and drop the new Emacs.app to /Applications, and on first opening from the Dock, I think i usually only see a message of the type "First time you launch this downloaded app, proceed?" (not sure about exact sentence and my system is French anyway).

I tried `codesign -dvv Emacs.app`, it works and appears to be simpler than a checksum check (which requires I maintain a template - with two spaces...-  where I edit the expected shasum, then I copy paste to Terminal window the whole thing, here I only need to copy paste the above, with a  `cd /Applications`; simple enough so ok for me to adapt this method, Thanks for advice!).
